packages/@aws-cdk/aws-lambda-event-sources/lib/stream.ts

```diff
@@ -78,6 +78,16 @@ export interface StreamEventSourceProps {
    */
   readonly maxBatchingWindow?: Duration;
 
+  /**
+   * The size of the tumbling windows to group records sent to DynamoDB or Kinesis
+   * Valid Range:
+   * * Minimum value of 0
+   * * Maximum value of 900
```

Comment:
```suggestion
   * Valid Range: 0 - 15 minutes
```

Reply:
This value is in seconds, CFN needs an int seconds. Should the valid range be in seconds to avoid confusion?

Reply:
What would be the confusion?
CDK expects a `Duration` class; it can be specified in any units. In the docs, I'd prefer it to be more human readable.

Reply:
Yep, all good have amended for humans ;)